src/tss2-sys/sysapi_util.c

```diff
@@ -304,3 +303,111 @@ int GetNumResponseHandles(TPM2_CC commandCode)
 {
     return GetNumHandles(commandCode, 0);
 }
+
+#ifdef DISABLE_WEAK_CRYPTO
+bool IsAlgorithWeak(TPM2_ALG_ID algorithm, TPM2_KEY_SIZE key_size)
```

Comment:
Shouldn't this be called `IsAlgorithmWeak`?

Reply:
Right, fixed.